src/pymortests/algorithms/ei.py

```diff
@@ -18,7 +19,9 @@ def test_ei_restricted_to_full(stationary_models):
     ei_op = EmpiricalInterpolatedOperator(op, collateral_basis=cb, interpolation_dofs=dofs, triangular=True)
     ei_model = model.with_(operator=ei_op)
 
-    for mu in model.parameters.space(1, 2).sample_randomly(3, seed=234):
+    with new_rng(234):
```

Comment:
Is there a reason why you turned the `seed=234` into the context manager block instead of omitting it completely like e.g. in thermalblock.py or parabolic_mor.py?

Reply:
For the tests, I basically wanted to change as little as possible compared to the current implementation. I believe in the long run, all those fixed seeds sprinkled across the test suite should be replaced by a better solution. (Using no initialization of the rng per test might not be the best solution, because then a test might start to fail (or pass) when a new test is executed before or some test is omitted.)

Reply:
I see. For now, I guess changing as little as possible is good. Maybe we should add an issue regarding the seeding strategy in the tests?

Reply:
My goto would be a function-scoped auto use fixture that sets the seed. However we also need to consider hypothesis, which has a mechanism for "foreign" RNGs IIRC.

Reply:
[Related](https://github.com/HypothesisWorks/hypothesis/issues/3131)

Reply:
I have created an issue: #1761